### PR TITLE
[SDCP-529] Fix various UI translations

### PR DESCRIPTION
**Pull request by MarkLark86**
* Patch strings in constant files after superdeskApi has initialised
* update planning.pot

### CODE REVIEW

File: client/components/UI/List/PubStatus.tsx [hunk 1]
```diff
@@ -2,24 +2,26 @@ import React from 'react';
...
  * @description Component to show published status of an item
  */
 export const PubStatus = ({item, isPublic}) => {
+    const {gettext} = superdeskApi.localization;
     let badge;
     let title = null;
 
     if (isPublic) {
-        title = gettext(TOOLTIPS.postedState);
+        title = gettext('Posted');
```
Comment (petrjasek) [inline client/components/UI/List/PubStatus.tsx:21]: why not use the tooltips here? seems like those are translated

Comment (MarkLark86) [reply client/components/UI/List/PubStatus.tsx:21]: That was a change from my first pass, before I got the Tooltips. I'll update that

File: po/planning.pot [hunk 2]
```diff
@@ -4,1791 +4,4338 @@ msgstr ""
...
+#: client/components/Planning/FeaturedPlanning/FeaturedPlanningModal.tsx:364
+msgid "All featured planning items are currently selected"
+msgstr ""
+
+#: client/actions/agenda.ts:42
+msgid "An agenda with this name already exists"
+msgstr ""
+
+#: client/components/ConfirmationModal.tsx:108
+msgid "Are you sure ?"
```
Comment (petrjasek) [inline po/planning.pot:376]: extra space

**Review by petrjasek — APPROVED** [review]
I'm not sure about that approach with assigning the translated labels later, imo it's a bit error prone as you can use the labels before those are translated. we can give it a try, maybe we can set some placeholder there rather then the actual string so it would be obvious if it's not used correctly

**Comment by MarkLark86** [comment]
These labels are used in React components, which are rendered by an AngularJS controller. So here we can guarantee they will be set for UI purposes.

I agree, it's not a good solution. Tomas has ideas we can discuss, maybe for 2.4?

**Comment by petrjasek** [comment]
sure

**Pull request merged**